appium/webdriver/extensions/location.py

```diff
@@ -29,13 +29,13 @@ def toggle_location_services(self):
         self.execute(Command.TOGGLE_LOCATION_SERVICES, {})
         return self
 
-    def set_location(self, latitude, longitude, altitude):
+    def set_location(self, latitude, longitude, altitude=None):
         """Set the location of the device
 
         Args:
             latitude (float): String or numeric value between -90.0 and 90.00
             longitude (float): String or numeric value between -180.0 and 180.0
-            altitude (float): String or numeric value
+            altitude (float, optional): String or numeric value
```

Comment:
Add comments that `altitude` is available only for Android real device. (need check for ios)

Reply:
> need check for ios

Not found implementations for altitude by searching with `setGeoLocation`, `altitude` under appium organization.